bin/propolis-server/src/lib/vm/mod.rs

```diff
@@ -137,6 +140,9 @@ pub(crate) type CrucibleReplaceResult =
 pub(crate) type CrucibleReplaceResultTx =
     oneshot::Sender<CrucibleReplaceResult>;
 
+/// PCI device instance ID type.
+type DeviceInstanceId = u32;
```

Comment:
I had (am having?) a little trouble following exactly what this type was (or is). It looks like it's an opaque identifier for the kstat series for a particular device; is that right? If so I think I would rename a bit, maybe to `KstatInstanceId`, and update the doc comment to denote that this is just a disambiguating value that might exist for any per-component kstat (it doesn't just have to be a PCI device).